### DISCUSSION TITLE
Can't access web UI from IP of machine but can access using localhost from the machine itself

### DISCUSSION THREAD

**Discussion by Nero10578** [bionic-gpt/bionic-gpt#346] [Announcements]
### Describe your issue

I don't understand why I can access the webui via localhose:7800 on the same machine but I can't access it from other machines using the ip address. I currently have it setup in a VM in my homelab so I'd like to be able to access it from other machines in my house...

### Describe your setup/startup scripts

I followed the install instructions using CPU and used the default docker compose. 

### Steps to reproduce

Try to access the webui from the machine ipaddress:7800 and it does not work.

### What was the expected result?

I expected to be able to access it the same as accessing it from localhose:7800

**Comment by tbendien** [comment]
Just go into the docker-compose.yml and do a word replace for localhost and enter the host IP. 

I think there are 7 places where it says localhost. Save the file, then bring Bionic up again.

**Reply by Nero10578** [reply]
Oh that's all. Thanks! Will try that.
